src/components/FeatureNotification/UserNotification.tsx

```diff
@@ -37,9 +40,18 @@ export default function UserNotification(): Notification | null {
     };
 
     const notifyTimeZoneUpdates = (userTz: InitializedTimeZone) => {
-      const notifyUser = userTz.timeZone && !userTz.timeZoneAcknowledgedOnMs;
+      const notifyUser =
+        userTz.timeZone &&
+        userTz.timeZoneNotificationCreatedMs &&
+        DateTime.now().plus({ days: -30 }).toMillis() <= userTz.timeZoneNotificationCreatedMs;
```

Comment:
this is fine but won't scale for org level stuff since we can't keep it on a per user preference..
should we just stop showing 30 days "after" user acknowledges the event? if so, then we can skip the created on preference... just a thought.

Reply:
We could do that, I thought the notification was removed after 30 days of creation, but maybe you are right, just use the acknowledge event